.eslintrc

```diff
@@ -0,0 +1,53 @@
+{
+  "extends": ["next", "prettier"],
+  "ignorePatterns": ["build/", "translations/"],
+  "globals": {
+    "fetch": false,
+    "Response": false,
+    "ReadableStream": false,
+    "DOMParser": false,
+    "IntersectionObserver": false,
+    "FormData": false
+  },
+  "env": {
+    "es6": true
+  },
+  "settings": {
+    "import/core-modules": [
+      "styled-jsx/css"
+    ]
+  },
+  "rules": {
+    "template-curly-spacing": "off",
+    "quotes": [2, "single"],
+    "semi": [2, "never"],
+    "indent": ["error", 2, {
+      "ignoredNodes": ["TemplateLiteral"],
+      // "SwitchCase": 1,
+      "ArrayExpression": 1,
+      "ObjectExpression": 1,
+      // "offsetTernaryExpressions": true,
+      "ImportDeclaration": 1
+
+    }],
+    "import/order": [2, {
+      "groups": ["builtin", "external", "internal", "parent", "sibling", "index", "object", "type"],
+      "newlines-between": "never"
+    }],
+    "padding-line-between-statements": [2,
+      { "blankLine": "always", "prev": "*", "next": "return" },
+      { "blankLine": "always", "prev": "block-like", "next": "*" }
+    ],
+    "@next/next/no-img-element": "off",
+    "react-hooks/rules-of-hooks": "warn",
+    "arrow-body-style": "off",
```

Comment:
I would recommend to set it to:
`"arrow-body-style": ["error", "as-needed"]`

See https://eslint.org/docs/latest/rules/arrow-body-style for more details